### PR TITLE
DAN | Full map update to 1950

### PR DESCRIPTION
**Pull request by danclave**

### CODE REVIEW

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_africa.txt [hunk 1]
```diff
@@ -789,4 +789,16 @@ KEN = {
 	
 	cultures = { kikuyu }
 	capital = STATE_KENYA
-}
\ No newline at end of file
+}
+
+# French West Africa
+FWA = {
```
Comment (settintotrieste) [inline 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_africa.txt:795]: French West Africa should use AOF to make it easier to transfer events by standardising tags between the V2 and V3 mods

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_africa.txt [hunk 2]
```diff
@@ -789,4 +789,16 @@ KEN = {
...
+	color = { 20  50  210 }
+
+	country_type = recognized
+
+	tier = colony
```
Comment (settintotrieste) [inline 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_africa.txt:800]: Federations containing multiple countries should use the ‘empire’ tier

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt [hunk 3]
```diff
@@ -562,4 +562,16 @@ BOU = { # Bougainville
 	
 	cultures = { melanesian }
 	capital = STATE_BOUGAINVILLE
+}
+
+# FRENCH INDO CHINA
+FIC = {
```
Comment (settintotrieste) [inline 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt:568]: Should use the ICU tag

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt [hunk 4]
```diff
@@ -562,4 +562,16 @@ BOU = { # Bougainville
...
+	color = { 20  50  210 }
+
+	country_type = recognized
+
+	tier = colony
```
Comment (settintotrieste) [inline 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt:573]: Should use the empire rage if the constituent states are kingdom, if not then kingdom if the constituent states are principalities

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/history/states/00_states.txt [hunk 5]
```diff
@@ -5763,7 +5762,7 @@ x83A6B2 xB82E36 }
 	}
 	s:STATE_ANHALT = {
 		create_state = {
-			country = c:GER
+			country = c:WGR
```
Comment (settintotrieste) [inline 1. CWE Base Mod/Cold War Era Mod (CWE)/common/history/states/00_states.txt:5765]: East Germany uses EGR. WGR is for West Germany

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt (hunk 4 above)
Comment (danclave) [reply 1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt:573]: Empire then, Cambodia etc were protectorate kingdoms

**Comment by settintotrieste** [comment]
Thanks for the edits! I’ll leave this branch open until you’re done with the Europe and Africa changes so let me know when you’re ready to merge.

File: 1. CWE Base Mod/Cold War Era Mod (CWE)/common/history/states/01_states_africa.txt [hunk 6]
```diff
@@ -0,0 +1,1233 @@
...
+			country = c:GBR
+			state_type = unincorporated
+			owned_provinces = { x162D0C x802111 xD6D784 x778011 xAF65D3 x51AD34 }
+		}	
+		
+		add_homeland = ewe
+		add_homeland = akan
+		add_homeland = fon
+	}
+	s:STATE_DAHOMEY = {
```
Comment (settintotrieste) [inline 1. CWE Base Mod/Cold War Era Mod (CWE)/common/history/states/01_states_africa.txt:81]: Should be owned by FRA

**Pull request merged**